JSDemos/Demos/DataGrid/ColumnTemplate/description.md

```diff
@@ -1 +1,3 @@
-The **DataGrid** widget supports custom column templates. In&nbsp;this demo, a&nbsp;custom column template is&nbsp;assigned to&nbsp;the **cellTemplate** option of&nbsp;the _Picture_ column.
\ No newline at end of file
+**DataGrid** columns support custom templates to display any content.
```

Comment:
I'm not sure that DataGrid **columns** can support anything. Maybe revert to the initial version? 

"The DataGrid supports custom templates for columns. You can use these templates to display any content."